tests/test_build.py

```diff
@@ -95,7 +105,7 @@ def test_toc_builds(tmpdir):
     assert "Unknown key in `_toc.yml`: foo" in err
 
 
-def test_build_errors(tmpdir):
+def test_build_errors(tmpdir, cli):
     # Create the book from the template
     path = Path(tmpdir).joinpath("mybook").absolute()
     run(f"jb create {path}".split())
```

Comment:
would this need to change to use the `CliRunner` as well?

Reply:
Yeah, I think everywhere the CLI is being invoked via `subprocess` should be using this Click facility instead.